File: appAdictive/src/main/java/appAdictive/dto/Estadisticas.java
package appAdictive.dto;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class Estadisticas {

	/**
	 * Cantidad de rutinas oficiales creadas esta semana
	 */
	private int oficiales;

	/**
	 * Cantidad de rutinas publicas creadas esta semana
	 */
	private int publicas;

	/**
	 * Cantidad de rutinas privadas creadas esta semana
	 */
	private int privadas;
	
	/**
	 * Cantidad de usuarios totales sin baneo
	 */
	private int baneado;
	
	/**
	 * Cantidad de usuarios totales baneados
	 */
	private int noBaneado;
}

File: deportivapp/src/app/models/estadisticas.ts
export class Estadisticas {

    oficiales:number;
	publicas:number;
	privadas:number;
	baneado:number;
	noBaneado:number;

    constructor(oficiales:number, publicas:number, privadas:number, baneado:number, noBaneado:number) {
        this.oficiales = oficiales;
        this.publicas = publicas;
        this.privadas = privadas;
        this.baneado = baneado;
        this.noBaneado = noBaneado;
    }
}